src/pages/Account/helpers.ts

```diff
@@ -10,7 +10,7 @@ import { QuestionsFormData } from "./Questions";
 
 type LocationUpdateData = {
   lastSeenAt: number;
-  lastSeenIn: { [key: string]: number } | null;
```

Comment:
This is not backward compatible, right? Will it break anything? You can consider changing the name of the attribute to keep backward incompatibility

Reply:
Changed in https://github.com/sparkletown/sparkle/pull/2202/commits/fa9ecad2e4a8b13737c78d565396fd369a53a2c1

Reply:
@mike-lvov A bit confused here:
`userLocation?.lastVenueIdSeenIn`

Is this going to be an `id` or is it going to be a path constructed of ids with slash as separator?

If it is the second, maybe the name is misleading and would be useful to rename as `lastVenuePath`, `lastLocation` or `lastPathSeenIn` or similar